docs/reference-guides/interactivity-api/README.md

```diff
@@ -40,15 +40,30 @@ You can start creating interactions once you set up a block development environm
 
 To indicate that the block [supports](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/) the Interactivity API features, add `"interactivity": true` to the `supports` attribute of the block's `block.json` file.
 
-```
+```json
+// block.json
 "supports": {
     "interactivity": true
 },
 ```
 
+Refer to the [`interactivity` support property docs](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-supports/#interactivity) to get a more detailed description of this property.
+
+#### Load Interactivity API Javascript code with `viewScriptModule`
+
+The Interactivity API script requires using Javascript modules that can be [natively loaded in WordPress from v6.5](https://make.wordpress.org/core/2024/03/04/script-modules-in-6-5/). The JavaScript files using the Interactivity API, should be enqueued via the [`viewScriptModule` block metadata](https://developer.wordpress.org/block-editor/reference-guides/block-api/block-metadata/#view-script-module):
+
+```json
+// block.json
+{
+   ...
+   "viewScriptModule": "file:./view.js"
+}
+```
+
 #### Add `wp-interactive` directive to a DOM element
 
-To "activate" the Interactivity API in a DOM element (and its children), add the [`wp-interactive`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-interactivity/packages-interactivity-api-reference/#wp-interactive) directive to the element in the block's `render.php` or `save.js` files.
+To "activate" the Interactivity API in a DOM element (and its children), add the [`wp-interactive`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-interactivity/packages-interactivity-api-reference/#wp-interactive) directive to the element in the block's `render.php` or `save.js` files. Its value must be the unique namespace of your plugin or block:
```

Comment:
I'm not sure this is accurate. The namespace is an Interactivity API concept. It _may_ match your plugin or block name. Or you may develop a plugin that shares the same namespace across many blocks.

I want to make sure we're not conflating a block name with an Interactivity API store - the namespaces are really about the stores and don't technically have anything to do with block names. They're unique store identifiers.

Reply:
Thanks for the explanation, @sirreal.

Can this directive be used without a value?
Example:
```
<div data-wp-interactive>
    <!-- Interactivity API zone -->
</div>
```

Reply:
Yes!

I just did some testing and it looks like `data-wp-interactive` is sufficient to recognize an interactive region.

Other stores can be accessed by prefixing the directive values with `namespace::` like this: `interactiveNamespace::state.whatever`. So if you have another store like `store( "otherNS", { state: { title: "Hello" } } )` you could do this:

```html
<div data-wp-interactive>
    <h1 data-wp-text="otherNS::state.title"></h1>
</div>
```

---

I went to this page to look it up: https://make.wordpress.org/core/2024/03/04/interactivity-api-dev-note/
I think the section "Working with other namespaces" should have described this but it looks like had some of the code samples broken. FYI @cbravobernal (I think the syntaxhighlighter block breaks HTML fairly often and it may be better to use the code block).

Reply:
Oks, I have removed the part about "Its value must be the unique namespace of your plugin or block:" as it's confusing and it seems too specific for these introductory steps.

Any additional explanations and clarifications should be added to the [`wp-interactive` documentation](https://developer.wordpress.org/block-editor/reference-guides/interactivity-api/api-reference/#wp-interactive)